terraform/certwatcher-lambda/main.tf

```diff
@@ -0,0 +1,104 @@
+
+# upload lambda zip to S3
+resource "aws_s3_bucket_object" "lambda-storage" {
+  bucket = var.s3_bucket
+  key    = var.s3_object
+  source = var.lambda_local_path
+  etag   = filemd5(var.lambda_local_path)
+}
```

Comment:
This could probably be done just as well with a local file upload. Uploading the binary to S3 doesn't seem to accomplish the same type of transparency or reproducibility as uploading javascript or python lambda code to S3 does.